app/graphql/types/payment_providers/adyen.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Types
+  module PaymentProviders
+    class Adyen < Types::BaseObject
+      graphql_name 'AdyenProvider'
+
+      field :api_key, String, null: false
+      field :hmac_key, String, null: true
+      field :id, ID, null: false
+      field :live_prefix, String, null: true
+      field :merchant_account, String, null: false
+
+      # NOTE: Api key is a sensitive information. It should not be sent back to the
+      #       front end application. Instead we send an obfuscated value
+      def api_key
+        "#{'•' * 8}…#{object.api_key[-3..]}"
+      end
+
+      def hmac_key
+        "#{'•' * 8}…#{object.hmac_key[-3..]}" if object.hmac_key.present?
+      end
```

Comment:
Is it possible to add unit tests for these two methods?

Reply:
I haven't found specs for graphql types in the project, but those methods are tested in `spec/graphql/mutations/payment_providers/adyen_spec.rb`